koku/api/report/queries.py

```diff
@@ -200,18 +202,40 @@ def get_query_param_data(self, dictkey, key, default=None):
             value = self.query_parameters.get(dictkey).get(key)
         return value
 
-    def get_order(self):
-        """Extract the order parameters and apply the associated ordering.
+    @property
+    def order_field(self):
+        """Order-by field name.
+
+        The default is 'total'
+        """
+        order_by = self.query_parameters.get('order_by', self.default_ordering)
+        return list(order_by.keys()).pop()
```

Comment:
Can we take more than one order by value? What if I want to order by account and service?